Index: utf8proc/CMakeLists.txt
===================================================================
--- utf8proc.orig/CMakeLists.txt
+++ utf8proc/CMakeLists.txt
@@ -68,7 +68,7 @@ endif()
 if(UTF8PROC_ENABLE_TESTING)
   enable_testing()
   file(MAKE_DIRECTORY data)
-  set(UNICODE_VERSION 15.0.0)
+  set(UNICODE_VERSION 15.1.0)
   file(DOWNLOAD https://www.unicode.org/Public/${UNICODE_VERSION}/ucd/NormalizationTest.txt ${CMAKE_BINARY_DIR}/data/NormalizationTest.txt SHOW_PROGRESS)
   file(DOWNLOAD https://www.unicode.org/Public/${UNICODE_VERSION}/ucd/auxiliary/GraphemeBreakTest.txt ${CMAKE_BINARY_DIR}/data/GraphemeBreakTest.txt SHOW_PROGRESS)
   add_executable(case test/tests.h test/tests.c utf8proc.h test/case.c)
Index: utf8proc/README.md
===================================================================
--- utf8proc.orig/README.md
+++ utf8proc/README.md
@@ -59,7 +59,7 @@ The C library is found in this directory
 and is named `libutf8proc.a` (for the static library) and
 `libutf8proc.so` (for the dynamic library).
 
-The Unicode version supported is 15.0.0.
+The Unicode version supported is 15.1.0.
 
 For Unicode normalizations, the following options are used:
 
Index: utf8proc/data/Makefile
===================================================================
--- utf8proc.orig/data/Makefile
+++ utf8proc/data/Makefile
@@ -22,7 +22,7 @@ CharWidths.txt: charwidths.jl EastAsianW
 	$(JULIA) charwidths.jl > $@
 
 # Unicode data version (must also update utf8proc_unicode_version function)
-UNICODE_VERSION=15.0.0
+UNICODE_VERSION=15.1.0
 
 UnicodeData.txt:
 	cp /usr/share/unicode/UnicodeData.txt $@
Index: utf8proc/utf8proc.c
===================================================================
--- utf8proc.orig/utf8proc.c
+++ utf8proc/utf8proc.c
@@ -101,7 +101,7 @@ UTF8PROC_DLLEXPORT const char *utf8proc_
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_unicode_version(void) {
-  return "15.0.0";
+  return "15.1.0";
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_errmsg(utf8proc_ssize_t errcode) {
