This patch is too simple to be described.
Index: utf8proc/utf8proc.c
===================================================================
--- utf8proc.orig/utf8proc.c
+++ utf8proc/utf8proc.c
@@ -101,7 +101,7 @@ UTF8PROC_DLLEXPORT const char *utf8proc_
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_unicode_version(void) {
-  return "12.0.0";
+  return "12.1.0";
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_errmsg(utf8proc_ssize_t errcode) {
Index: utf8proc/data/Makefile
===================================================================
--- utf8proc.orig/data/Makefile
+++ utf8proc/data/Makefile
@@ -22,7 +22,7 @@ CharWidths.txt: charwidths.jl EastAsianW
 	$(JULIA) charwidths.jl > $@
 
 # Unicode data version (must also update utf8proc_unicode_version function)
-UNICODE_VERSION=12.0.0
+UNICODE_VERSION=12.1.0
 
 UnicodeData.txt:
 	cp /usr/share/unicode/UnicodeData.txt $@
Index: utf8proc/README.md
===================================================================
--- utf8proc.orig/README.md
+++ utf8proc/README.md
@@ -39,7 +39,7 @@ The C library is found in this directory
 and is named `libutf8proc.a` (for the static library) and
 `libutf8proc.so` (for the dynamic library).
 
-The Unicode version supported is 12.0.0.
+The Unicode version supported is 12.1.0.
 
 For Unicode normalizations, the following options are used:
 
