This patch is too simple to be described.
Index: utf8proc/utf8proc.c
===================================================================
--- utf8proc.orig/utf8proc.c
+++ utf8proc/utf8proc.c
@@ -101,7 +101,7 @@ UTF8PROC_DLLEXPORT const char *utf8proc_
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_unicode_version(void) {
-  return "12.0.0";
+  return "12.1.0";
 }
 
 UTF8PROC_DLLEXPORT const char *utf8proc_errmsg(utf8proc_ssize_t errcode) {
Index: utf8proc/data/Makefile
===================================================================
--- utf8proc.orig/data/Makefile
+++ utf8proc/data/Makefile
@@ -22,7 +22,7 @@ CharWidths.txt: charwidths.jl EastAsianW
 	$(JULIA) charwidths.jl > $@
 
 # Unicode data version (must also update utf8proc_unicode_version function)
-UNICODE_VERSION=12.0.0
+UNICODE_VERSION=12.1.0
 
 UnicodeData.txt:
 	cp /usr/share/unicode/UnicodeData.txt $@
Index: utf8proc/README.md
===================================================================
--- utf8proc.orig/README.md
+++ utf8proc/README.md
@@ -39,7 +39,7 @@ The C library is found in this directory
 and is named `libutf8proc.a` (for the static library) and
 `libutf8proc.so` (for the dynamic library).
 
-The Unicode version supported is 12.0.0.
+The Unicode version supported is 12.1.0.
 
 For Unicode normalizations, the following options are used:
 
Index: utf8proc/utf8proc.h
===================================================================
--- utf8proc.orig/utf8proc.h
+++ utf8proc/utf8proc.h
@@ -71,7 +71,7 @@
 /** The MAJOR version number (increased when backwards API compatibility is broken). */
 #define UTF8PROC_VERSION_MAJOR 2
 /** The MINOR version number (increased when new functionality is added in a backwards-compatible manner). */
-#define UTF8PROC_VERSION_MINOR 2
+#define UTF8PROC_VERSION_MINOR 3
 /** The PATCH version (increased for fixes that do not change the API). */
 #define UTF8PROC_VERSION_PATCH 0
 /** @} */
Index: utf8proc/Makefile
===================================================================
--- utf8proc.orig/Makefile
+++ utf8proc/Makefile
@@ -21,7 +21,7 @@ UCFLAGS = $(CPPFLAGS) $(CFLAGS) $(PICFLA
 # The API version number is defined in utf8proc.h.
 # Be sure to also update these ABI versions in MANIFEST and CMakeLists.txt!
 MAJOR=2
-MINOR=2
+MINOR=3
 PATCH=0
 
 OS := $(shell uname)
